### DISCUSSION TITLE
[Suggest] What do you need when running the moleculerjs in the production

### DISCUSSION THREAD

**Discussion by lucwj** [moleculerjs/educational-content#8] [Ideas]
# This is a guide helping you to write an article

> `moleculerjs` is the awesome micro-services framework, so, when we run our project in production, there are many other kinds of stuff that we need to run along with `moleculerjs`

Your article should have the following structure:

## Structure

- Title
- Introduction (briefly explain what people will learn in this article)
- Goals (explain what does the person "gain" by reading the article/tutorial)
- Prerequisites (mention the software versions that are required and the necessary knowledge background)
- Step-by-step instructions (explain how to achieve the above mentioned goal)
- Provide a link to the source code of your article/tutorial (e.g., github)
- Conclusion

**Comment by AndreMaz** [comment]
Sorry, currently this topic is not on our priority list and, therefore, it not covered by the community program. However, we would love to hear your testimonials and expertise about this topic.
